src/NLog.MailKit/MailTarget.cs

```diff
@@ -522,16 +522,40 @@ private MimeMessage CreateMailMessage(LogEventInfo lastEvent, string body)
 
             if (Priority != null)
             {
-                var renderedPriority = Priority.Render(lastEvent);
-                try
-                {
+                // default is already set by MimeMessage ctor
+                //msg.Priority = MessagePriority.Normal;
+
+                string renderedPriority = Priority.Render(lastEvent);
 
-                    msg.Priority = (MessagePriority)Enum.Parse(typeof(MessagePriority), renderedPriority, true);
+                // try backward compatible enum:System.Net.Mail.MailPriority
+                if (ConversionHelpers.TryParseEnum(renderedPriority, out System.Net.Mail.MailPriority mailPriority))
+                {
+                    // only care for Low/High
+                    switch (mailPriority)
+                    {
+                        case System.Net.Mail.MailPriority.High:
+                            {
+                                InternalLogger.Info("Compatibility: converting [System.Net.Mail.MailPriority.High] to [MimeKit.MessagePriority.Urgent]. Consider updating your configuration!");
```

Comment:
If we support this case, I don't think these internal logs are needed. Maybe as debug otherwise?